Game/include/MapTypes.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef MAPTYPES_HPP
+#define MAPTYPES_HPP
+
+enum MAP_TYPES
```

Comment:
You can use an enum class instead.
enumerator names are local to the enum and their values do not implicitly convert to other types (like another enum or int)